digiwf-engine/digiwf-engine-service/src/main/resources/prozesse/example/digiwf-kennenlernen/element-templates/BasicUsertask.json

```diff
@@ -0,0 +1,161 @@
+      "type": "Dropdown",
+      "value": "false",
+      "choices": [
+        { "name": "Ja", "value": "true" },
+        { "name": "Nein", "value": "false" }
+      ],
+      "binding": {
+        "type": "camunda:inputParameter",
+        "name": "app_notification_send_candidate_groups"
+      }
+    },
+    {
+      "label": "Candidate Users benachrichtigen?",
+      "description": "Sendet eine E-Mail an die Candidate Users",
+      "type": "Dropdown",
+      "value": "false",
+      "choices": [
+        { "name": "Ja", "value": "true" },
+        { "name": "Nein", "value": "false" }
+      ],
+      "binding": {
+        "type": "camunda:inputParameter",
+        "name": "app_notification_send_candidate_users"
+      }
+    },
+    {
+      "label": "Assignee benachrichtigen?",
+      "description": "Sendet eine E-Mail an den Assginee",
+      "type": "Dropdown",
+      "value": "false",
+      "choices": [
+        { "name": "Ja", "value": "true" },
+        { "name": "Nein", "value": "false" }
+      ],
+      "binding": {
+        "type": "camunda:inputParameter",
+        "name": "app_notification_send_assignee"
+      }
+    },
+    {
+      "label": "Individueller Betreff",
+      "type": "String",
+      "description": "Wird kein Betreff angegeben, wird der Standart Betreff verwendet",
+      "binding": {
+        "type": "camunda:inputParameter",
+        "name": "mail_subject"
+      }
+    },
+    {
+      "label": "Individueller E-Mail Text",
+      "description": "Wird kein E-Mail Text angegeben, wird der Standart E-Mail Text verwendet",
+      "type": "Text",
+      "binding": {
+        "type": "camunda:inputParameter",
+        "name": "mail_body"
+      }
+    },
+    {
+      "label": "Individueller E-Mail Gruß",
+      "description": "Wird kein E-Mail Gruß angegeben, wird der Standart E-Mail Gruß verwendet",
+      "type": "Text",
+      "binding": {
+        "type": "camunda:inputParameter",
+        "name": "mail_bottom_text"
+      }
+    },
+    {
+      "label": "User für Vorgang berechtigen?",
+      "description": "Gibt einem Nutzer nach Abschluss der Aufgabe Zugriff auf die Vorgangsübersicht",
+      "type": "Dropdown",
+      "value": "false",
+      "choices": [
+        { "name": "Ja", "value": "true" },
+        { "name": "Nein", "value": "false" }
+      ],
+      "binding": {
+        "type": "camunda:inputParameter",
+        "name": "app_assign_user_to_processinstance"
+      }
+    },
+    {
+      "label": "S3 Pfade für Schreibrechtigungen",
+      "description": "; separierte Liste an S3 Pfaden",
+      "type": "String",
+      "binding": {
+        "type": "camunda:inputParameter",
+        "name": "app_file_paths"
+      }
+    },
+    {
+      "label": "S3 Pfade für Lesebrechtigungen",
+      "description": "; separierte Liste an S3 Pfaden",
+      "type": "String",
+      "binding": {
+        "type": "camunda:inputParameter",
+        "name": "app_file_paths_readonly"
+      }
+    }
+  ]
+}
```

Comment:
Das ist das Gleiche, wie https://github.com/it-at-m/digiwf-core/blob/4288bc25d26dd9b6f7ab5b638a0a7a475841f675/docs/src/.vuepress/public/element-template/user-task-template.json